docs/rc.xml.all

```diff
@@ -374,4 +374,15 @@
     </device>
   </libinput>
 
+  <!--
+  <windowRules>
+    <windowRule identifier="" event="">
+      <action name="">
+    </windowRules>
+    <windowRule identifier="">
+      <serverDecoration></serverDecoration>
+    </windowRules>
+  </windowRules>
```

Comment:
Having this example completely empty feels a bit strange, I'd suggest filling it with something which makes it easier to understand by the context on the first look, especially if somebody didn't read the man page.